larq_compute_engine/mlir/transforms/prepare_tf.cc

```diff
@@ -17,10 +18,21 @@ using compute_engine::core::bitpacking_bitwidth;
 
 // Prepare LCE operations in functions for subsequent legalization.
 struct PrepareLCE : public PassWrapper<PrepareLCE, FunctionPass> {
+  PrepareLCE() = default;
+  PrepareLCE(const PrepareLCE& pass) {}
+  PrepareLCE(const LCETarget target) {
+    target_arm_.setValue(target == LCETarget::ARM);
+  }
+
   void runOnFunction() override;
   void getDependentDialects(DialectRegistry& registry) const override {
     registry.insert<mlir::lq::LarqDialect>();
   }
+
+ private:
+  Option<bool> target_arm_{
+      *this, "target-arm",
+      llvm::cl::desc("Include patterns to target the ARM platform.")};
```

Comment:
Great 👍 

How do you feel about making this an enum?
```cpp
  Option<LCETarget> target_{
      *this, "target",
      llvm::cl::desc("Platform to target."),
      llvm::cl::values(
          clEnumValN(LCETarget::ARM, "arm", "ARM target"),
          clEnumValN(LCETarget::ARM, "xmos", "XMOS target"))};
```
which can then be configured using `-tfl-prepare-lce=target=arm`.

But since we only have two targets, I am also happy to keep it as a boolean for now.